lib/licenses.nix

```diff
@@ -649,6 +649,12 @@ lib.mapAttrs (n: v: v // { shortName = n; }) {
     url = http://metadata.ftp-master.debian.org/changelogs/main/d/debianutils/debianutils_4.8.1_copyright;
   };
 
+  stk = {
```

Comment:
Please put this into a separate commit with the message ``licenses: add stk``.

Reply:
Thanks for the review.
All done.